src/client/src/components/form/formUtils.ts

```diff
@@ -29,21 +29,12 @@ export const parseFloatWithThousandsSeparator = (numericString?: string) => {
   return parseFloat(numericString.replace(/'/g, ""));
 };
 
-/**
- * Get the maximum precision from two numeric strings.
- * @param {string} numericString1 The first string.
- * @param {string} numericString2 The second string.
- * @returns The maximum precision from the two strings.
- */
-export const getMaxPrecision = (numericString1: string, numericString2: string) =>
-  Math.max(getPrecisionFromString(numericString1), getPrecisionFromString(numericString2));
-
 /**
  * Get the precision from a string.
  * @param {string} numericString The string to get the precision from.
  * @returns The precision of the string.
  */
-export const getPrecisionFromString = (numericString: string) => numericString.split(".")[1]?.length || 0;
+export const getDecimalsFromNumericString = (numericString: string) => numericString.split(".")[1]?.length || 0;
```

Comment:
PP: Das funktioniert nicht, wenn der `numericString` in scientific notation daherkommt eventuell diese Einschränkung in Doc-Comment erwähnen.